docs/source/user-guide/integrations/spark_databricks.rst

```diff
@@ -0,0 +1,170 @@
+Spark & Databrick
+=================
+
+This tutorial demonstrates how deepchecks can be used on the Databricks ML platform using Spark. We will build a
+logistic regression model on top of the Adult dataset, a sample dataset that is automatically available on every
+databricks workspace.
+
+Loading the dataset
+-------------------
+We first define the dataset schema and then load it as a Spark dataframe.
+
+.. code-block:: python
+
+    schema = """`age` DOUBLE,
+    `workclass` STRING,
+    `fnlwgt` DOUBLE,
+    `education` STRING,
+    `education_num` DOUBLE,
+    `marital_status` STRING,
+    `occupation` STRING,
+    `relationship` STRING,
+    `race` STRING,
+    `sex` STRING,
+    `capital_gain` DOUBLE,
+    `capital_loss` DOUBLE,
+    `hours_per_week` DOUBLE,
+    `native_country` STRING,
+    `income` STRING"""
+
+    dataset = spark.read.csv("/databricks-datasets/adult/adult.data", schema=schema)
+
+    # Splitting the data to train/test set
+    trainDF, testDF = dataset.randomSplit([0.8, 0.2], seed=42)
+
+Defining deepchecks Dataset
+---------------------------
+
+.. code-block:: python
+
+    from deepchecks.tabular import Dataset
+
+    pd_train = trainDF.toPandas()
```

Comment:
perhaps note here that sampling is recommended and is OK logically as most checks sample anyway?